Translate the social media section

nscyclone opened the pull request:
This refers to #453.

devleoper commented on _config.yml:
```diff
@@ -138,3 +138,22 @@ toc:
+  my:
+  pl:
+  no:
+  ko:
```
In Korean, "이 페이지 공유하기" can be proper.

3100 replied:
In Japanese, It would be "このページをシェアする".

therobinsj replied:
In Norwegian, "Del denne siden"

michalvalasek replied:
In Slovak: "Zdieľaj"

julienXX replied:
In French: "Partager cette page".

cizixs replied:
In Chinese: 分享本页面

christiansakai replied:
@nscyclone in ID, it is "Bagikan Halaman Ini"

nscyclone replied:
Thanks, @christiansakai, I've just added it.

nscyclone commented:
Thanks, @devleoper, I've just added your translation.

nscyclone commented:
I've just mentioned some translators here. Sorry for bothering but I hope you don't mind to spend a minute to help us complete this.

nscyclone commented:
Thanks, @3100!

nscyclone commented:
Thanks, @therobinsj!

doomspork commented:
Thank you for doing this @nscyclone!!

nscyclone commented:
Thanks, @michalvalasek and @julienXX!

nscyclone commented:
Thanks, @cizixs, I've just added your translation.

devleoper commented on _includes/social.html:
```diff
@@ -5,7 +5,8 @@
 {% endif %}
 <hr />
-<h3 id="social">Share This Page</h3>
+{% assign lang = page.lang || site.default_lang %}
```
There is exactly the same code [at line 2](https://github.com/nscyclone/elixir-school/blob/18e45569ebb5397235bc3f7d50f0e525ad2ef1a7/_includes/social.html#L2).

How about moving this line at the top of this file, and remove the other line which does same work?

nscyclone commented:
Thanks, @devleoper, I've just fixed that!

I've also moved the task list from this PR to the [corresponding issue](https://github.com/doomspork/elixir-school/issues/453). Probably we could merge this because there is almost no translation progress for a significant amount of time. What do you think about this, @doomspork ?

doomspork commented:
I'm happy to merge this, should we use English for the missing translations?

> On Jul 17, 2016, at 3:25 PM, Dmitry <notifications@github.com> wrote:
> 
> Thanks, @devleoper, I've just fixed that!
> 
> I've also moved the task list from this PR to the corresponding issue. Probably we could merge this because there is almost no translation progress for a significant amount of time. What do you think about this, @doomspork ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

nscyclone commented:
@doomspork, yes, sure, English is used when no translation is provided in the config.

doomspork commented:
Great!  This is good to merge, we can update the remainder as we go.

> On Jul 17, 2016, at 5:27 PM, Dmitry <notifications@github.com> wrote:
> 
> @doomspork, yes, sure, English is used when no translation is provided in the config.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

christiansakai commented:
@nscyclone in ID, it is "Bagikan Halaman Ini"

nscyclone commented:
10/15 translations done. Thanks everyone!
